### PR TITLE
Initial Mocha tests 

### PR DESCRIPTION
**Pull request by lakindu95**
#14

### CODE REVIEW

File: test/indexTest.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  });
+});
+
+//Testing the dynamodb options function
+describe("Check the dynamodb function", function(){
+  it("Region should be localhost",function(){
+    expect((dynamodbOptions,{region: "localhost"})).to.deep.include({region: "localhost"});
+  });
+
+  it("Should listen to endpoint", function () {
```
Comment (rehrumesh) [inline test/indexTest.js]: This block is no needed

File: test/indexTest.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+//Test cases to check the get port function
+describe("Port",function(){
+  it("Port should return number", function(){    
+    let myport = dataApp.prototype.port;   
+    assert(typeof myport, 'number');
+  });
+
+  it("Port value should be >= 0 and < 65536", function () {
+    http.get("http://localhost:8000", function (response) {
```
Comment (rehrumesh) [inline test/indexTest.js]: Is it the correct test? This doesn't match with the description

Comment (lakindu95) [reply test/indexTest.js]: will check

File: test/indexTest.js (hunk 1 above)
Comment (lakindu95) [reply test/indexTest.js]: Removed the block

File: test/indexTest.js [hunk 3]
```diff
@@ -1,24 +1,81 @@
...
-            console.log('after');
-        });
+//Define the modules required to mocha testing
+const assert = require("chai").assert;
+const http = require ("http");
+const expect = require("chai").expect;
+const should = require("should");
+const aws = require ("aws-sdk");
+const seeder = require("../src/seeder.js");
+const dataApp = require("../index.js");
```
Comment (AshanFernando) [inline test/indexTest.js:9]: You can rename the variable as dynamodbLocal

File: test/testTable [hunk 4]
```diff
@@ -0,0 +1,112 @@
+//Define the modules required to mocha testing
```
Comment (rehrumesh) [inline test/testTable]: shouldn't this file be a JS file?

**Pull request merged**